strawberry/types/fields/resolver.py

```diff
@@ -190,6 +187,18 @@ def __call__(self, *args: str, **kwargs: Any) -> T:
     def signature(self) -> inspect.Signature:
         return Signature.from_callable(self._unbound_wrapped_func, follow_wrapped=True)
 
+    # TODO: find better name
+    @cached_property
+    def strawberry_annotations(
+        self,
+    ) -> Dict[inspect.Parameter, Union[StrawberryAnnotation, None]]:
+        return {
+            p: StrawberryAnnotation(p.annotation, namespace=self._namespace)
+            if p.annotation is not inspect.Signature.empty
+            else None
```

Comment:
A small nit here, otherwise the alignment makes it harder to read

```suggestion
            p: (
                StrawberryAnnotation(p.annotation, namespace=self._namespace)
                if p.annotation is not inspect.Signature.empty
                else None
            )
```